docker_sigprofiler/command_line.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import argparse
+from sigproextractor import sigpro as sig
+
+
+def main():
+    '''
+    please pass parameters as required.
+    '''
+    parser = argparse.ArgumentParser(
+        description='This script is implemented for accept parameters for the SigProfiler.'
+    )
+
+    parser.add_argument(
+        '-i', '--input',
+        dest='input_file',
+        action='store',
```

Comment:
'store' is the default value, so you're safe to remove this line, also the line in other args.